core/src/test/java/com/google/cloud/sql/core/SqlAdminApiFetcherTest.java

```diff
@@ -1,200 +0,0 @@
-/*
```

Comment:
Did we mean to delete this test? Seems like having good unit tests around our API interactions would be good.

Reply:
Yes, these all were moved to CloudSqlInstanceTest.

Reply:
But should they be? CloudSqlInstance is a wholly different class with different responsibilities.

Reply:
I moved the test back to SqlAdminApiFetcherTest